### DISCUSSION TITLE
How to move to a new system on the same machine?

### DISCUSSION THREAD

**Discussion by wunderbaum** [immich-app/immich#14416] [Q&A] [answered]
As my SDD quickly get to small for all my photos, I want to reinstall it with a bigger harddisk. Can I just take out the old sdd, install a clean system on the new sdd and then copy the whole directory images from the old sdd to the new one?

**Comment by bo0tzz** [comment, marked as answer]
It's best to do a full backup and then restore that, see https://immich.app/docs/administration/backup-and-restore.

**Reply by wunderbaum** [reply]
Thanks!  I just did this and worked smoothly!
